chosen.js

```diff
@@ -30,7 +30,7 @@
       return {
         restrict: 'A',
         require: '?ngModel',
-        terminal: true,
+        priority: 1,
```

Comment:
Removal of terminal allows ngAria to function. Need priority > 0 to allow angular-chosen ngModel definition to avoid being overwritten by angular 1.4.9+'s postLink re-definition